demo_modules/gears/gears_client.js

```diff
@@ -19,6 +19,19 @@ export function load(debug, state, wallGeometry) {
 
       // A map of teeth -> Path2D.
       this.gearPaths_ = [];
+
+      this.gearsState = state.define('gears', {
+        x: a => a,
```

Comment:
So no interpolators?

Reply:
There weren't any here originally. Gears doesn't really need the 'state' abstraction; it was just a more reliable way to do the networking.